common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -129,21 +131,16 @@ object WhiskConfig {
    * Reads a Map of key-value pairs from the environment (sys.env) -- store them in the
    * mutable properties object.
    */
-  def readPropertiesFromFile(properties: scala.collection.mutable.Map[String, String], file: File)(
-    implicit logging: Logging) = {
+  def readPropertiesFromFile(properties: scala.collection.mutable.Map[String, String], file: File) = {
     if (file != null && file.exists) {
-      logging.info(this, s"reading properties from file $file")
       for (line <- Source.fromFile(file).getLines if line.trim != "") {
         val parts = line.split('=')
         if (parts.length >= 1) {
           val p = parts(0).trim
           val v = if (parts.length == 2) parts(1).trim else ""
           if (properties.contains(p)) {
             properties += p -> v
-            logging.debug(this, s"properties file set value for $p")
           }
-        } else {
-          logging.warn(this, s"ignoring properties $line")
```

Comment:
Do we lose anything by removing this? Like debuggability of failed startups?

Reply:
I think once we needs to debug the config, one can lookup the properties in the env vars in the container.

Reply:
I disagree - there's been many occasions when there were mismatches in the config parameters, deployment fails, and one doesn't know why.